client/src/components/StepsQuestions/index.css

```diff
@@ -0,0 +1,74 @@
+/* ================================= */
+/* Basic style */
+/* ================================= */
+
+.steps {
+  margin-top: 4rem;
+  padding: 3rem 0;
+}
+
+.steps-content {
+  min-height: 40rem;
+  margin-top: 16px;
+  border: 1px dashed #e9e9e9;
+  border-radius: 6px;
+  background-color: #fafafa;
+  padding: 1.6rem 0.4rem;
+}
+
+.title {
+  text-align: start;
+  font-size: 1.6rem;
+  margin-bottom: 3rem;
+}
+
+.img__box {
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  margin-bottom: 3rem;
+}
+
+.img__box--img {
+  width: 90%;
+}
+
+.input {
```

Comment:
Also here, .input is a generic / common class name so it would be a good idea to make this .input selector more specific: `.steps .input`